index.bs

```diff
@@ -5943,6 +5994,15 @@ Each stored [=virtual authenticator=] has the following properties:
     [=User Verification=] will always succeed. If set to [FALSE], it will fail.
 
     Note: This property has no effect if |hasUserVerification| is set to [FALSE].
+: |extensions|
+:: A string array containing the [=extension identifiers=] supported by the [=Virtual Authenticator=].
+
+    A [=Virtual authenticator=] MUST support all [=authenticator extensions=] present in its |extensions| array.
+    It MUST NOT support any [=authenticator extension=] not present in its |extensions| array.
+: |uvm|
+:: A {{UvmEntries}} array to be set as the [=authenticator extension output=] when processing the [=User Verification Method=] extension.
+
+   Note: This property has no effect if the [=Virtual Authenticator=] does not support the [=User Verification Method=] extension.
```

Comment:
Heh, this needs one more space of indentation to render as intended.

```suggestion
    Note: This property has no effect if the [=Virtual Authenticator=] does not support the [=User Verification Method=] extension.
```

Reply:
Something something meaningful indentation is evil. Done!